alerts/apps_alerts.libsonnet

```diff
@@ -24,6 +24,23 @@
             'for': '15m',
             alert: 'KubePodCrashLooping',
           },
+          {
+            expr: |||
+              (
+                kube_pod_container_status_last_terminated_reason{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s, reason="OOMKilled"} == 1
+              ) and on(pod) (
+                increase(kube_pod_container_status_restarts_total{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s}[5m]) > 0
```

Comment:
Since you are only matching the pod label here, the second operand will be true whenever one container from this pod is restarted, no matter the reason. So, if one of the containers was last terminated because it was OOMKilled long ago, and another container restart for whatever reason, this alert will fire.
I'd suggest matching both the pod and its namespace to make sure that we are matching the correct pod here and not one from another namespace, and the container to ensure that the alert does not fire for other containers restarts.